### PR TITLE
replace netstandard1.6 with netstandard2.0

### PR DESCRIPTION
**Pull request by tylerl0706**
need to test on Windows

### CODE REVIEW

File: PowerShellEditorServices.build.ps1 [hunk 1]
```diff
@@ -18,7 +18,7 @@ param(
 
 $script:IsCIBuild = $env:APPVEYOR -ne $null
 $script:IsUnix = $PSVersionTable.PSEdition -and $PSVersionTable.PSEdition -eq "Core" -and !$IsWindows
-$script:TargetFrameworksParam = "/p:TargetFrameworks=\`"$(if (!$script:IsUnix) { "net451;" })netstandard1.6\`""
+$script:TargetFrameworksParam = "/p:TargetFrameworks=\`"$(if (!$script:IsUnix) { "net451;" })netstandard2.0\`""
```
[the remark this answers is not in the document]
Comment (SeeminglyScience) [reply PowerShellEditorServices.build.ps1]: @rkeithhill For now I think so.  The question is what to do about the shims.  We could either ship `netstandard.dll` or require .NET 4.7.1 to be installed.  I don't feel particularly good about either of those, but changing the .NET requirement would probably need to wait until 2.0 (assuming that's even an option)

[the remark this answers is not in the document]
Comment (tylerl0706) [reply PowerShellEditorServices.build.ps1]: Keep in mind, this is a change that will go into master.

Is is not a part of v2

[the remark this answers is not in the document]
Comment (rkeithhill) [reply PowerShellEditorServices.build.ps1]: > It is not a part of v2
yeah, I was not grokking that bit.

**Comment by tylerl0706** [comment]
I don't think we need this because @rjmholt's got a PR out to move everything

**Pull request closed without merging**